### ISSUE TITLE
Labels not set

### ISSUE THREAD

**Issue by mhubig** [cloudfoundry-community/terraform-provider-cloudfoundry#480]
Hi there, im trying to set labels on a bunch of PCF Apps and it it seems to succeed but the labels are missing after apply.

```shell
$ cf api
API endpoint:   https://api.sys.prod.gcp.xxx.cloud
API version:    3.128.0
```
Example cloudfoundry_app code snippet:

```terraform
resource "cloudfoundry_app" "service" {
  name       = "${var.service_name}-${var.system_environment}"
  space      = var.system_space
  instances  = var.service_instances
  memory     = var.service_memory
  disk_quota = var.service_disk_quota
  timeout    = 360
  strategy   = "rolling"

  health_check_http_endpoint = var.health_check_http_endpoint
  health_check_type          = var.health_check_http_endpoint != null ? "http" : null
  health_check_timeout       = var.health_check_http_endpoint != null ? 180 : null

  buildpacks = var.buildpacks
  path       = data.artifactory_file.service.output_path

  labels = {
    "tags.datadoghq.com/scope"                  = var.service_datadog_scope
    "tags.datadoghq.com/env"                    = var.system_environment
    "tags.datadoghq.com/service"                = var.service_name
    "tags.datadoghq.com/version"                = var.service_version
    "tags.datadoghq.com/product_billing_number" = var.service_datadog_billing_number
  }

  environment = merge({
    TZ = "Europe/Berlin"

    JBP_CONFIG_OPEN_JDK_JRE                = "{ jre: { version: 17.+ }}"
    JBP_CONFIG_SPRING_AUTO_RECONFIGURATION = "{ enabled: false }"
    JBP_CONFIG_METRIC_WRITER               = "{ enabled: false }"

    JSON_LOGGING            = "true"
    STD_LOG_COLLECTION_PORT = 10514
    LOGS_CONFIG             = "[{\"type\":\"tcp\",\"port\":\"10514\",\"source\":\"java\",\"service\":\"base-data-service\"}]"

    JAVA_OPTS = "-XX:MaxDirectMemorySize=1G -Dio.netty.allocator.numDirectArenas=0 -Dio.netty.noPreferDirect=true"

    DD_TAGS                         = "team:${var.service_datadog_scope},scope:${var.service_datadog_scope},env:${var.system_environment},service:${var.service_name},version:${var.service_version},product_billing_number:${var.service_datadog_billing_number}"
    DD_ENV                          = var.system_environment
    DD_SERVICE                      = var.service_name
    DD_VERSION                      = var.service_version
    DD_LOG_LEVEL                    = "warn"
    DD_LOGS_INJECTION               = true
    DD_JMXFETCH_ENABLED             = true
    DD_TRACE_SPAN_TAGS              = "env:${var.system_environment}"
    DD_HTTP_CLIENT_TAG_QUERY_STRING = true
    DD_HTTP_SERVER_TAG_QUERY_STRING = true
  }, var.service_environment_variables)

  stopped = var.stopped

  routes {
    route = cloudfoundry_route.external.id
  }

  routes {
    route = cloudfoundry_route.internal.id
  }

  dynamic "routes" {
    for_each = var.service_additional_routes
    content {
      route = routes.value
    }
  }

}

```

**Comment by Thanhphan1147** [comment]
Hello, indeed the labels attribute is not taken into account by the v3appdeployer for now, so for the current version of the provider setting them won't have any effect

**Comment by Thanhphan1147** [comment]
Should be fixed with PR #494
